extensions/amp-video/validator-amp-video.protoascii

```diff
@@ -69,6 +69,8 @@ attr_lists: {
     name: "noaudio"
     value: ""
   }
+  attrs: { name: "object-fit" }
```

Comment:
Is it safe to update the validation and the code in one PR, since they roll out separately?  In theory, couldn't validation end up rolling out before the code does (which, admittedly, wouldn't be that bad, since it would just be an undocumented attribute that does nothing anyway)

Reply:
I've seen this concern before and honestly I'm not too worried about it. At the worst is someone's page is valid before something fully works. That shouldn't be too much of a time delta. Validation gets pushed daily/weekly and runtime weekly (with an additional week for canary). So at worse it's a week or two from proper usage. I haven't heard of any complaints for validation being ahead of runtime but if it becomes an issue lets address it when it does become one.